style.css

```diff
@@ -524,3 +524,78 @@ main{
     background-color: var(--secondary-grey);
     opacity: 0.9;
 }
+
+/* Input */
+.input-outlined{
+    padding: 0.8rem 0.6rem;
+    border: 1px solid  var(--secondary-grey);
+    border-radius: 5px;
+}
+
+.input-outlined:hover{
+    border: 1px solid  gray;
+}
+
+.input-outlined:focus{
+    outline: none;
+    border: 1px solid var(--primary-maroon);
+}
+
+.input-filled{
+    padding: 0.8rem 0.6rem;
+    border-left: none;
+    border-right: none;
+    border-top: none;
+    border-bottom: 1px solid;
```

Comment:
You could have also done border: none and border-bottom: 1px solid black, would have given the same result.